src/org/zaproxy/zap/extension/httpsinfo/ZapAddOn.xml

```diff
@@ -2,16 +2,19 @@
 	<name>HttpsInfo</name>
 	<version>7</version>
 	<status>alpha</status>
-	<description>Adds Https status information</description>
+	<description>Adds a context menu to the site tree and history table use of 
+	which displays HTTPS status information, scans for and 
+	alerts on some HTTPS vulnerabilities.</description>
```

Comment:
Hmmm ok. This is something we may need to evaluate going forward in a more broad scope.

I had someone ask me yesterday "how do you use extensions"...which isn't consistent or obvious (nor should it necessarily be), so we kind of need a way to tell users.

I can reduce this back to the short entry that it previously was.